### ISSUE TITLE
Make a minimal end-to-end pipeline

### ISSUE THREAD

**Issue by aimalz** [alice-crafford/Photo-z-Stress-Test#4]
This issue is for isolating a minimal subset of the Golden Spike notebook in this repository to enable #5. It should include:

1. Preparing a creator model with pzflow (can be read in from the example file for now).
2. Drawing 10,000-galaxy and 100,000-galaxy catalogs of photometry and redshift.
3. Degrading the larger sample with the LSST photometric error model to make a test set.
4. Degrading the smaller sample with the inverse redshift incompleteness degrader to make a training set.
5. Informing/training the FlexZBoost estimator on the training set.
6. Performing estimation with the FlexZBoost estimator on the test set.
7. Evaluating some metrics from the Golden Spike.

**Comment by alice-crafford** [comment]
I think I get to close this! I wanted to make sure though, before I actually do that :)
